README.MD

```diff
@@ -65,6 +65,14 @@ mongo_db_bundle:
         ssl: false
         connectTimeoutMS: 3000 # default null (no timeout)
         readPreference: primaryPreferred # see https://docs.mongodb.com/manual/reference/read-preference/#primary for info
+      
+      ssl_client_name:    # see https://www.php.net/manual/en/mongo.connecting.ssl.php 
+        uri: 'mongodb://host1:3062,host2'
+        password: ''
+        authSource: ''
+        ssl: true
+        driver_options:
+            context: "app.mongodb.context_service" # default null see https://symfony.com/doc/master/bundles/DoctrineMongoDBBundle/config.html#specifying-a-context-service
```

Comment:
What's this for? Aren't we confusing stuff with a different bundle?

Reply:
In this section we can see an example hiow to implement a service which creates the correct context stream. But I can remove it and add an example in the docs where you thinks is the right place.

Reply:
I have looked at the offical doctrine documentation on at symfony.com to try to do things as symfony way as possible. So, I build a solution for the driver options like doctrine does.

Service example:
app.mongodb.context_service:
        class: 'resource'
        factory: 'stream_context_create'
        arguments:
            - { ssl: { verify_expiry: true } }